### ISSUE TITLE
How can i set different configuration per staging environment?

### ISSUE THREAD

**Issue by JoostKiens** [kaliberjs/build#37]
It is not clear how to use different sets of configurations based on staging environment/NODE_ENV.

Could we use something like [https://www.npmjs.com/package/config](https://www.npmjs.com/package/config)?

**Comment by EECOLOR** [comment]
It will not be based on `NODE_ENV` but `CONFIG_ENV`: https://www.npmjs.com/package/@kaliber/config

I have not had the time to add it so that it would work on both client and server. We probably need to introduce a naming convention in our configuration, something like:

```
module.exports = {
  client: ...
  server: ...
}
```

**Comment by klaasman** [comment]
We can probably use the `@kaliber/config` package on the server and pass client-config via the `*.html.js` to the universal components, e.g.:

```js
import App from 'App?universal'
import config from '@kaliber/config'

return () => (
  <html>
    <head />
    <body>
      <App config={config.client} />
    </body>
  </html>
)

**Comment by klaasman** [comment]
Fixed in cc07a9975845a7015bd2ab46f2df35fcb9eda4eb
